### PR TITLE
Fix linting issues

### PR DESCRIPTION
**Pull request by TheiLLeniumStudios**
**Describe Pull request**
First, make sure you've read and are following the contribution guidelines and style guide and your code reflects that.
Write up a clear and concise description of what your pull request adds or fixes and if it's an added feature explain why you think it should be included in the core.

If your PR is to fix an issue mention that issue here

**Questions (please complete the following information):**
- Have you personally loaded this code into an updated qbcore project and checked all it's functionality? [yes/no] (Be honest)
- Does your code fit the style guidelines? [yes/no]
- Does your PR fit the contribution guidelines? [yes/no]

### CODE REVIEW

File: client/client.lua [hunk 1]
```diff
@@ -100,7 +100,7 @@ CreateThread(function()
 end)
 
 CreateThread(function()
-    local hour = 0
+    local hour
```
Comment (IdrisDose) [inline client/client.lua:103]: ?? why was this removed?

File: server/server.lua [hunk 2]
```diff
@@ -202,7 +202,7 @@ end)
 
 -- COMMANDS
 
-RegisterCommand('freezetime', function(source, args, rawCommand)
+RegisterCommand('freezetime', function(source, _, _)
```
Comment (IdrisDose) [inline server/server.lua:205]: You can remove the '_'

File: server/server.lua [hunk 3]
```diff
@@ -215,7 +215,7 @@ RegisterCommand('freezetime', function(source, args, rawCommand)
     TriggerClientEvent('QBCore:Notify', source, Lang:t('error.not_allowed'), 'error')
 end)
 
-RegisterCommand('freezeweather', function(source, args, rawCommand)
+RegisterCommand('freezeweather', function(source, _, _)
```
Comment (IdrisDose) [inline server/server.lua:218]: You can remove the '_'

File: server/server.lua [hunk 4]
```diff
@@ -228,7 +228,7 @@ RegisterCommand('freezeweather', function(source, args, rawCommand)
     TriggerClientEvent('QBCore:Notify', source, Lang:t('error.not_allowed'), 'error')
 end)
 
-RegisterCommand('weather', function(source, args, rawCommand)
+RegisterCommand('weather', function(source, args, _)
```
Comment (IdrisDose) [inline server/server.lua:231]: You can remove the '_'

File: server/server.lua [hunk 5]
```diff
@@ -246,7 +246,7 @@ RegisterCommand('weather', function(source, args, rawCommand)
     end
 end)
 
-RegisterCommand('blackout', function(source, args, rawCommand)
+RegisterCommand('blackout', function(source, _, _)
```
Comment (IdrisDose) [inline server/server.lua:249]: You can remove the '_'

File: server/server.lua [hunk 6]
```diff
@@ -260,7 +260,7 @@ RegisterCommand('blackout', function(source, args, rawCommand)
     TriggerClientEvent('QBCore:Notify', src, Lang:t('error.not_allowed'), 'error')
 end)
 
-RegisterCommand('morning', function(source, args, rawCommand)
+RegisterCommand('morning', function(source, _, _)
```
Comment (IdrisDose) [inline server/server.lua:263]: You can remove the '_'

File: server/server.lua [hunk 7]
```diff
@@ -269,7 +269,7 @@ RegisterCommand('morning', function(source, args, rawCommand)
     end
 end)
 
-RegisterCommand('noon', function(source, args, rawCommand)
+RegisterCommand('noon', function(source, _, _)
```
Comment (IdrisDose) [inline server/server.lua:272]: You can remove the '_'

File: server/server.lua [hunk 8]
```diff
@@ -278,7 +278,7 @@ RegisterCommand('noon', function(source, args, rawCommand)
     end
 end)
 
-RegisterCommand('evening', function(source, args, rawCommand)
+RegisterCommand('evening', function(source, _, _)
```
Comment (IdrisDose) [inline server/server.lua:281]: You can remove the '_'

File: server/server.lua [hunk 9]
```diff
@@ -287,7 +287,7 @@ RegisterCommand('evening', function(source, args, rawCommand)
     end
 end)
 
-RegisterCommand('night', function(source, args, rawCommand)
+RegisterCommand('night', function(source, _, _)
```
Comment (IdrisDose) [inline server/server.lua:290]: You can remove the '_'

File: server/server.lua [hunk 10]
```diff
@@ -296,7 +296,7 @@ RegisterCommand('night', function(source, args, rawCommand)
     end
 end)
 
-RegisterCommand('time', function(source, args, rawCommand)
+RegisterCommand('time', function(source, args, _)
```
Comment (IdrisDose) [inline server/server.lua:299]: You can remove the '_'

**Review by IdrisDose — CHANGES_REQUESTED** [review]
Please remove the underscores.

File: server/server.lua (hunk 2 above)
Comment (BerkieBb) [reply server/server.lua:205]:
```suggestion
RegisterCommand('freezetime', function(source)
```

File: server/server.lua (hunk 3 above)
Comment (BerkieBb) [reply server/server.lua:218]:
```suggestion
RegisterCommand('freezeweather', function(source)
```

File: server/server.lua (hunk 4 above)
Comment (BerkieBb) [reply server/server.lua:231]:
```suggestion
RegisterCommand('weather', function(source, args)
```

File: server/server.lua (hunk 5 above)
Comment (BerkieBb) [reply server/server.lua:249]:
```suggestion
RegisterCommand('blackout', function(source)
```

File: server/server.lua (hunk 6 above)
Comment (BerkieBb) [reply server/server.lua:263]:
```suggestion
RegisterCommand('morning', function(source)
```

File: server/server.lua (hunk 7 above)
Comment (BerkieBb) [reply server/server.lua:272]:
```suggestion
RegisterCommand('noon', function(source)
```

File: server/server.lua (hunk 8 above)
Comment (BerkieBb) [reply server/server.lua:281]:
```suggestion
RegisterCommand('evening', function(source)
```

File: server/server.lua (hunk 9 above)
Comment (BerkieBb) [reply server/server.lua:290]:
```suggestion
RegisterCommand('night', function(source)
```

File: server/server.lua (hunk 10 above)
Comment (BerkieBb) [reply server/server.lua:299]:
```suggestion
RegisterCommand('time', function(source, args)
```
